src/dotnet/commands/dotnet-test/TestRunners/TestRunner.cs

```diff
@@ -51,9 +51,10 @@ private ICommand CreateTestRunnerCommand()
             var commandArgs = _argumentsBuilder.BuildArguments();
 
             return _commandFactory.Create(
-                _testRunner,
+                $"dotnet-{_testRunner}",
                 commandArgs,
-                new NuGetFramework("DNXCore", Version.Parse("5.0")));
+                new NuGetFramework("DNXCore", Version.Parse("5.0")),
+                Constants.DefaultConfiguration);
```

Comment:
That's becase in that case we are not using the port and the designtime option. We are not running them through VS. We did the work for running on the console. It is a really small fix and I will get to it soon enough, but I want to unblock debugging tests through VS and then I will do the work to fully support all of params (-o, -b, -r, -f, -c) for both the console and VS. Makes sense?

If you look at the history of the code, when running design time, we were not passing the config to ExecuteRunnerCommand.